### ISSUE TITLE
can't install activeAdmin gem with rails 5 - api version - actionpack dependency conflict

### ISSUE THREAD

**Issue by elanperach** [activeadmin/activeadmin#4613]
```
Bundler could not find compatible versions for gem "actionpack":
  In Gemfile:
    rails (~> 5.0) was resolved to 5.0.0, which depends on
      actionmailer (= 5.0.0) was resolved to 5.0.0, which depends on
        actionpack (= 5.0.0)

    rails (~> 5.0) was resolved to 5.0.0, which depends on
      actionmailer (= 5.0.0) was resolved to 5.0.0, which depends on
        actionpack (= 5.0.0)

    activeadmin was resolved to 1.0.0.pre4, which depends on
      inherited_resources (~> 1.6) was resolved to 1.6.0, which depends on
        actionpack (< 5, >= 3.2)

```

**Comment by elanperach** [comment]
after adding the inherited_resources gem  it worked

```
gem 'inherited_resources', github: 'activeadmin/inherited_resources'
gem 'activeadmin', '~> 1.0.0.pre4'
```

**Comment by dangodev** [comment]
@elanperach nice fix! Worked for me on Ruby 2.4.0.
